### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by TouKoua**
A branch I was using to test Bootstrap installations. This is the current working version of the project with the basic initial webpage. Should have everything you need to get started.

### CODE REVIEW

File: README.md
Comment (TouKoua) [inline README.md:1]: Updated ReadMe with Instructions

File: Richmond_Library_App/__pycache__/__init__.cpython-311.pyc
Comment (TouKoua) [inline Richmond_Library_App/__pycache__/__init__.cpython-311.pyc:1]: Remove compile file

File: Richmond_Library_App/__pycache__/settings.cpython-311.pyc
Comment (TouKoua) [inline Richmond_Library_App/__pycache__/settings.cpython-311.pyc:1]: Remove compile file

File: Richmond_Library_App/__pycache__/wsgi.cpython-311.pyc
Comment (TouKoua) [inline Richmond_Library_App/__pycache__/wsgi.cpython-311.pyc:1]: Remove compile file

File: Richmond_Library_App/__pycache__/urls.cpython-311.pyc
Comment (TouKoua) [inline Richmond_Library_App/__pycache__/urls.cpython-311.pyc:1]: Remove compile file

File: Richmond_Library_App/settings.py
Comment (TouKoua) [inline Richmond_Library_App/settings.py:1]: Added Bootstrap 5 to the installed app

File: Richmond_Library_App/templates/home.html
Comment (TouKoua) [inline Richmond_Library_App/templates/home.html:1]: Added bootstrap to the html.

File: Richmond_Library_App/views/__pycache__/home.cpython-311.pyc
Comment (TouKoua) [inline Richmond_Library_App/views/__pycache__/home.cpython-311.pyc:1]: remove compile file

**Comment by nobodyname12** [comment]
Looks good! I'm excited to get started

**Review by nobodyname12 — APPROVED** [review]

**Review by ale202008 — APPROVED** [review]
Looks great.

**Pull request merged**